README.md

```diff
@@ -21,6 +24,38 @@ Barnabas provides a way to run an [Apache Kafka][kafka] cluster on
 
 <!-- /TOC -->
 
+## Deploying Kubernetes/OpenShift cluster
+
+A Kafka/Zookeeper cluster can be deployed on a Kubernete or OpenShift cluster. So, first of all, such deployment is needed.
+
+### Kubernetes
+
+In order to interact with a Kubernetes cluster, be sure to have [the kuberctl tool][kubectl] installed.
+If you don't have a Kubernetes cluster up and running, the simpler way to deploy it, for development purpose, is to use the [Minikube][minikube] project which provides a one node cluster in a VM. In order to do that you can just follow the installation guide which describes all the prerequisites and how to get the Minikube binaries.
```

Comment:
Wouldn't something as *easiest* be better than *simpler*? Again, just a feeling from a non native speaker.

Reply:
Agree